script/Ciel/Day 14/14_01A_CIEL14A_5.txt

```diff
@@ -721,22 +778,26 @@ We both wanted this time of parting to last even longer.
 [sha:ec3c9cfbe267a0c08e4cacb49b0abd16979c538a]{
 -- Page 202, Offset 38225.
 -- いやな男。最後まで憎まれ口をたたくんだから。ちょっとぐらい優しいコト、言ってくれるかなって期待したのに」
-What an awful man. You've been saying rude things to me even to the very end. I was hoping that maybe you would say something a little nice to me for once."
+You really are awful.
+ You're still being mean to me, right up until the very end.
+ I'd been hoping you might actually say something nice to me for once."
 }
 [sha:e8348d9479f8b333b6953f0bbfb26efabd351921]{
 -- Page 202, Offset 38226.
 -- 「なんだそりゃ。優しくしてほしかったら、もっとか弱いトコ見せろ、ばか」
-"What the hell. If you wanted me to be nice, then you should have loosened up more, you idiot."
+"What the hell.
+ If you wanted me to be nice to you, you should have opened up more, you idiot."
 }
 [sha:0e9011ea5cc6ae8ca5b8536ed591d902107c11e7]{
 -- Page 203, Offset 38227.
 -- お望み通り、心の底から本音を返す。
-I return the feelings from the bottom of my heart as I please.
+Since she asked nicely,
+ I'll tell her how I really feel.
 }
 [sha:20cbf855689d684b6a1b11ea5103c4ff3459254e]{
 -- Page 203, Offset 38228.
 -- ……いつかの夜を思い出す。
-...I think back to a night some time ago.
+I think back on the other night...
 }
 [sha:3bad97dde6f03a184912cbcfdaff959f7ca9cb20]{
 -- Page 203, Offset 38229.
```

Comment:
Let's at least have some fun with this exceptionally rare word. It also just feels like the PERFECT usage of "gallivant", to be honest.
This can also connect much more directly.

>When the two of us gallivanted around in the city together.